modules/services/picom.nix

```diff
@@ -269,6 +271,21 @@ in {
       '';
     };
 
+    extraWintypes = mkOption {
+      type = types.str;
+      default = "";
+      example = ''
```

Comment:
```suggestion
      example = literalExample ''
```

Reply:
I'm kinda new to this, so I was wondering, what's the difference between just using the string and using the string with `literalExample` (I just copied the format for this option from the `extraConfig` option)? Thanks!

Reply:
without `literalExample`, it gets rendered badly: https://rycee.gitlab.io/home-manager/options.html#opt-services.picom.extraOptions (note the extra `''`s)

With `literalExample`, you get something like: https://rycee.gitlab.io/home-manager/options.html#opt-services.pantalaimon.settings which looks much better.

Reply:
Oh I see, so I'll add literalExample for extraConfig too then